FantasmoSDK/Classes/Network/FMApi.swift

```diff
@@ -168,6 +168,16 @@ class FMApi {
             error: postError)
     }
     
+    
+    func send(arKitTrackingStateStatistics: ARKitTrackingStateStatistics) {
```

Comment:
We do not know the API situation yet. This should not be part of this PR.

Reply:
Ok, I have removed the method